src/components/ChatBotBody/ChatBotBody.tsx

```diff
@@ -59,7 +59,17 @@ const ChatBotBody = ({
 	const bodyStyle: CSSProperties = {
 		...styles?.bodyStyle,
 		scrollbarWidth: settings.chatWindow?.showScrollbar ? "auto" : "none",
-	}
+	};
+
+	// styles for typing indicator
+	const rcbTypingIndicatorStyle: CSSProperties = {
+		...styles?.rcbTypingIndicatorStyle
+	};
+
+	// styles for rcb dot
+	const rcbDotStyle: CSSProperties = {
+		...styles?.rcbDotStyle
+	};
```

Comment:
One last nitpick - since there are currently no additional inline styles, you can include these directly in the element instead of defining constants here.

Reply:
I pushed the changes.